src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java

```diff
@@ -280,14 +281,18 @@ private void tickPluginToInstall(PluginSpec spec) {
         // the target plugin web element becomes stale due to the dynamic behaviour of the plugin
         // manager UI which ends up with StaleElementReferenceException.
         // This is re-trying until the element can be properly checked.
-        waitFor().withTimeout(Duration.ofSeconds(10)).until(() -> {
-            try {
-                check(find(by.xpath("//input[starts-with(@name,'plugin.%s.')]", name)));
-            } catch (NoSuchElementException | StaleElementReferenceException e) {
-                return false;
-            }
-            return true;
-        });
+        try {
```

Comment:
I don;t think this is the change you need though - it is the "check for updates" that is throwing the issue isn't it?

or at least requires a corresponding fix there.

Reply:
The check for updates is in production code. This is the point at which the test code fails (as seen in PR description).

Reply:
> The check for updates is in production code ?

 This is the point at which the test code fails (as seen in PR description).
 
Then I think you got passed the first bit that would cause a failure (actually getting the json).